conf/modules/structural_variant_calling.config

```diff
@@ -19,6 +19,9 @@ process {
     */
 
     withName: '.*:STRUCTURAL_VARIANT_CALLING:SNIFFLES' {
+
+        ext.prefix = { "${meta.id}.sniffles" }
```

Comment:
Question about `.` and `_` usage in the pipeline. Are you consistently using the `.` syntax to separate parts of a filename? In the raredisease pipe we try to use `_` whenever possible and reserve `.` for file endings. Both works but it would be good to be consistent within the pipeline

Reply:
Let's change it to using `_`.